engine/src/main/java/org/terasology/engine/entitySystem/metadata/EventMetadata.java

```diff
@@ -38,7 +38,8 @@ public EventMetadata(Class<T> simpleClass, CopyStrategyLibrary copyStrategies, R
             skipInstigator = simpleClass.getAnnotation(BroadcastEvent.class).skipInstigator();
         }
         if (networkEventType != NetworkEventType.NONE && !isConstructable() && !Modifier.isAbstract(simpleClass.getModifiers())) {
-            logger.error("Event '{}' is a network event but lacks a default constructor - will not be replicated", this);
+            logger.atError().addArgument(() -> this).
+                    log("Event '{}' is a network event but lacks a default constructor - will not be replicated");
```

Comment:
Simple reference of current object.
```suggestion
            logger.error("Event '{}' is a network event but lacks a default constructor - will not be replicated", this); //NOPMD
```